ledfx/color.py

```
@@ -70,6 +71,46 @@ def __init__(self, colors, mode="linear", angle="90"):
+        if pos >= last_pos:
+            return "#{:02x}{:02x}{:02x}".format(
+                last_color.red,
+                last_color.green,
+                last_color.blue,
+            )
+
+        # Find containing segment and linearly interpolate
+        for (c1, p1), (c2, p2) in zip(stops, stops[1:]):
+            if p1 <= pos <= p2:
+                t = 0.0 if p2 == p1 else (pos - p1) / (p2 - p1)
+                r = int(round(c1.red + (c2.red - c1.red) * t))
```

Ruff (0.12.2)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)